packages/protobuf-test/src/gen/ts/google/protobuf/descriptor_pb.ts

```diff
@@ -164,12 +164,21 @@ export class FileDescriptorProto extends Message<FileDescriptorProto> {
 
   /**
    * The syntax of the proto file.
-   * The supported values are "proto2" and "proto3".
+   * The supported values are "proto2", "proto3", and "editions".
```

Comment:
Are we already handling this in descriptor-set.ts and in protoplugin?

Reply:
We were.  I also added a small commentary on this in the main description.